wifimgr.py

```diff
@@ -119,62 +222,58 @@ def handle_root(client):
-                            <td>Password:</td>
-                            <td><input name="password" type="password" /></td>
-                        </tr>
-                    </tbody>
-                </table>
-                <p style="text-align: center;">
-                    <input type="submit" value="Submit" />
-                </p>
-            </form>
-            <p>&nbsp;</p>
-            <hr />
-            <h5>
-                <span style="color: #ff0000;">
-                    Your ssid and password information will be saved into the
-                    "%(filename)s" file in your ESP module for future usage.
-                    Be careful about security!
-                </span>
-            </h5>
-            <hr />
-            <h2 style="color: #2e6c80;">
-                Some useful infos:
-            </h2>
-            <ul>
-                <li>
-                    Original code from <a href="https://github.com/cpopp/MicroPythonSamples"
-                        target="_blank" rel="noopener">cpopp/MicroPythonSamples</a>.
-                </li>
-                <li>
-                    This code available at <a href="https://github.com/tayfunulu/WiFiManager"
-                        target="_blank" rel="noopener">tayfunulu/WiFiManager</a>.
-                </li>
-            </ul>
+                        <li onclick="document.getElementById('ssid').focus()">
+                            <span class="fixedWidth">SSID</span>
+                            <input id="ssid" name="ssid" type="text"/>
+                        </li>
+                        <li onclick="document.getElementById('password').focus()">
+                            <span class="fixedWidth">Password</span>
+                            <input id="password" name="password" type="password"/>
+                        </li>
+                        <li onclick="document.getElementById('submit').click()">
+                            <input id=submit type="submit" value="Submit" />
+                        </li>
+                    </ul>
+                    <div class="placeholder"></div>
+                    <h2>
+                        Infos:
+                    </h2>
+                    <p>
+                        Your ssid and password information will be saved into the "{}" file in your ESP module for future usage. Be careful about security!
+                    </p>
+                    <p>
+                        Original code from <a href="https://github.com/cpopp/MicroPythonSamples" target="_blank" rel="noopener">cpopp/MicroPythonSamples</a>.
+                    </p>
+                    <p>
+                        This code available at <a href="https://github.com/tayfunulu/WiFiManager" target="_blank" rel="noopener">tayfunulu/WiFiManager</a>.
+                    </p>
+                </form>
+            </body>
         </html>
-    """ % dict(filename=NETWORK_PROFILES))
+    """.format(NETWORK_PROFILES))
```

Comment:
please do not change named placeholders to positional placeholders in non-short/non-trivial cases.